cfy_manager/components/rabbitmq/scripts/start_rabbitmq_server.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+# call "rabbitmqctl shutdown" when exiting
+trap "{ echo Stopping rabbitmq; /usr/sbin/rabbitmqctl shutdown; exit 0; }" EXIT
+
+echo "Starting rabbitmq"
+/usr/sbin/rabbitmq-server
```

Comment:
Minor nitpick: It's nicer to have a newline on the end of the file.

Reply:
Done